src/state_machine/coordinator/fire.rs

```diff
@@ -568,7 +568,7 @@ impl<Aggregator: AggregatorTrait> Coordinator<Aggregator> {
         }
 
         let mut dkg_failures = HashMap::new();
-
+        let threshold: usize = self.config.threshold.try_into().unwrap();
```

Comment:
Can't we use `?` instead of `unwrap`?

Reply:
Only if we add TryFromInt to the coordinator Error, which is a major api rev. I added it on the other PR.